packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -2193,6 +2193,23 @@ class ShaderMaskLayer extends ContainerLayer {
   }
 }
 
+/// A backdrop key uniquely identifies the backdrop that a [BackdropFilterLayer]
+/// samples from.
+///
+/// When multiple backdrop filters share the same key, the Flutter engine can
+/// more efficiently perform the backdrop operations.
+///
+/// For more information, see `BackdropFilter`.
```

Comment:
```suggestion
/// For more information, see [BackdropFilter].
```

Reply:
Done

Reply:
This says "done" but it wasn't change yet.